lib/components/note-preview/index.tsx

```diff
@@ -117,7 +117,7 @@ export const NotePreview: FunctionComponent<Props> = ({
           let matchCount = 0;
 
           const content = note.content.replace(
-            /(- \[x\]|- \[ \])/g,
+            /^(- \[x\])|(- \[ \])/gim,
```

Comment:
this only allows checkboxes to appear at the beginning of the line but they only need to be the first non-whitespace characters on a line. check out how it's done in the utility function

https://github.com/Automattic/simplenote-electron/blob/8b6b30197c1a3d6ffa7b5cacc9a17b1157b6ee29/lib/utils/task-transform.ts#L1-L6

Reply:
Checkboxes in lists are not valid markdown nor is it the way it works on Android or iOS. I am going to modify that logic to unify it with the way the mobile clients work.